src/Bridges/ApplicationLatte/TemplateFactory.php

```diff
@@ -112,4 +119,10 @@ public function createTemplate(UI\Control $control = NULL)
 		return $template;
 	}
 
+	public function changeTemplateClass($templateClass)
+	{
+		\Nette\Utils\Validators::assert($templateClass, 'string');
```

Comment:
How would you prevent passing something different than string? I understand your note about consistency, but it's wrong I think. Or should I remove usage of `Validators` and use custom check?